setup.py

```diff
@@ -17,7 +17,7 @@
 setup(
     # Information
     name='faster-fifo',
-    version='0.0.2',
+    version='1.0.0',
```

Comment:
Is this the only change in the commit?
Why I don't see the change in the C++ file?
Please don't be hasty

Reply:
No, it wasn't that. It's just that the fix yesterday was correct. But the push to pip happened without the previous commit where I had removed that line. So technically, I just rebuilt it again by changing the log version i.e the commit today and pushed it to PyPI. You can do a 'pip install faster-fifo' and check. I wouldn't commit anything without checking it works.